### PR TITLE
Remove unused meeting session status codes from the meeting event gui…

### PR DESCRIPTION
**Pull request by simmkyu**
…de and improve explanations

**Issue #:**
N/A

**Description of changes:**
- Remove unused meeting session status codes from the meeting event guide
- Add descriptions for `SignalingBadRequest` and improve error descriptions

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Review the generated HTML files
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? No
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
...
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Removed
+
+### Fixed
+- Remove unused meeting session status codes from the meeting event guide and add more error messages
```
Comment (devalevenkatesh) [inline CHANGELOG.md:17]: Should we move this to "removed" section :) ?

File: guides/06_Meeting_Events.md [hunk 2]
```diff
@@ -200,10 +212,14 @@ The following table shows common error messages you may receive when failing to
 
 |Messages|Status code|Suggested resolution
 |--|--|--
+|The attendee couldn't join because the meeting was at capacity.|AudioCallAtCapacity|Ensure that the number of attendees does not exceed the maximum limit (250). See the [Amazon Chime SDK quotas](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). You can also track the number of attendees in your server application using the [Amazon Chime SDK event notifications](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-notifications.html).
 |The meeting already ended.|MeetingEnded|Ensure that you or someone else have not deleted a meeting using the [DeleteMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteMeeting.html) API action in your server application. A meeting also automatically ends after a period of inactivity. See the [Chime SDK developer guide](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html) for details.
+|The signaling connection was closed with code (close code) and reason: (reason)|SignalingBadRequest|Ensure that you do not use the deleted attendee's response in the `DefaultMeetingSession` object to start the meeting.
```
Comment (devalevenkatesh) [inline guides/06_Meeting_Events.md:217]: We have one more case for `SignalingBadRequest`:

Builders may also use the `CreateAttendee` response from a different (older auto-ended / deleted) `CreateMeeting` response and then they will run into the `SignalingBadRequest` error. This case can happen if the same `ClientRequestToken` is used in `CreateMeeting` API and the older meeting ends and builders get a new meeting response but they still use the older attendee response which won't work with new meeting join. So, checking if we should add this case as well here?

File: CHANGELOG.md (hunk 1 above)
Comment (simmkyu) [reply CHANGELOG.md:17]: A good point - I'll update the description as I have more fixes.

File: guides/06_Meeting_Events.md (hunk 2 above)
Comment (simmkyu) [reply guides/06_Meeting_Events.md:217]: Nice, I didn't know that. I'll add a description.

File: guides/06_Meeting_Events.md [hunk 3]
```diff
@@ -200,10 +212,14 @@ The following table shows common error messages you may receive when failing to
...
 |The meeting already ended.|MeetingEnded|Ensure that you or someone else have not deleted a meeting using the [DeleteMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteMeeting.html) API action in your server application. A meeting also automatically ends after a period of inactivity. See the [Chime SDK developer guide](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html) for details.
+|The signaling connection was closed with code (close code) and reason: (reason)|SignalingBadRequest|Ensure that you do not use the deleted attendee's response in the `DefaultMeetingSession` object to start the meeting. You also should not use the attendee response from the finished meeting that you created with the same `ClientRequestToken` parameter before.
```
Comment (devalevenkatesh) [inline guides/06_Meeting_Events.md:217]: Should we say "ended" meeting than "finished" meeting?

Comment (simmkyu) [reply guides/06_Meeting_Events.md]: Thanks for the suggestion! I wasn't sure between "ended" and "finished" too.

Comment (devalevenkatesh) [reply guides/06_Meeting_Events.md]: cool, thanks for updating!

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**